modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json

```
@@ -821,6 +821,16 @@
         "1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","X"
       ]
     },
+    {
+      "id": "macaca_mulatta_EVA",
+      "source_name": "EVA",
+      "description": "Variants from EVA",
+      "species": "macaca_mulatta",
+      "assembly": "Mmul_10",
+      "type": "remote",
+      "use_as_source" : 1,
+      "filename_template" : "http://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_3/by_species/macaca_mulatta/GCA_003339765.3/GCA_003339765.3_current_ids.vcf.gz"
```

Thanks @dglemos! Just a small suggestion: please use HTTPS here.

```suggestion
      "filename_template" : "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_3/by_species/macaca_mulatta/GCA_003339765.3/GCA_003339765.3_current_ids.vcf.gz"
```